### ISSUE TITLE
make all mod formables to visaul JEs so ppl know they exits

### ISSUE THREAD

**Issue by xEiweix** [MasterSon-git/vic3-kmpmp#153]
for example Iberia

**Comment by Tziplays** [comment]
????

**Comment by xEiweix** [comment]
man sieht das iberien formable bzw. die anforderungen nicht in den JEs

**Comment by Yukinu** [comment]
Es gibt ein JE für das Formen von Iberia?

**Comment by Yukinu** [comment]
Byzanz, Polen, Win (west Indies and Jamaica), Iberia, Arabien, Groß Kolumbien, Mali, TRH (Usbekistans), Indien, BHT (Belutschistan), Caucasus, KAZ (Uralsk), ALD (Algier), PEU (Lima), UNL(Niederlande), RHN (Rheinbund), CLI (N&South Carolina), PLT (Buenos Aires), PAK(Kashmir), Java, Indonesien, Indien, GER, ITA, SCA, FND

das wären die vom Basegame die ich in den Datei gefunden habe

Gemodded kommt noch P01 dazu, was auch immer das ist.

Ich würde allerdings nicht alle übernehmen wollen muss ich ehrlich sagen oder wie seht ihr das? Habe ich noch welche vergessen?

**Comment by MasterSon-git** [comment]
P01 ist Parthia, das formable für Persien, ich hatte gehofft nen generischen JE machen zu können damit wir das nicht für jedes Land/Formable einzeln machen müssen, hab aber keine Ahnung ob das geht.

**Comment by MasterSon-git** [comment]
add this as new type of JE "Guide"

**Comment by Yukinu** [comment]
Just an idea to generate a JE for the nations that have a fromable to point to the JE tab to check for the possible formables?

would be easy to maintain in case paradox changes the code

**Comment by MasterSon-git** [comment]
@Yukinu Yeah i think thats the way to go, if possible we should list all possible formables add a link and show the conditions how to form each of them.

The JEs mentioned on top:
Leader of Latin America JE
Sugar Monopoly
Ruler of the East
Master of the Med

Aren't formables, so they can be ignored - @xEiweix what was the reason to list these JEs in this task?
